package.json

```diff
@@ -1,26 +1,26 @@
 {
   "name": "pdf.js",
   "devDependencies": {
-    "@babel/core": "^7.22.11",
-    "@babel/plugin-transform-modules-commonjs": "^7.22.11",
-    "@babel/preset-env": "^7.22.14",
-    "@babel/runtime": "^7.22.11",
+    "@babel/core": "^7.22.17",
+    "@babel/plugin-transform-modules-commonjs": "^7.22.15",
+    "@babel/preset-env": "^7.22.15",
+    "@babel/runtime": "^7.22.15",
     "@javascript-obfuscator/escodegen": "2.3.0",
     "acorn": "^8.10.0",
     "autoprefixer": "^10.4.15",
     "babel-loader": "^9.1.3",
-    "caniuse-lite": "^1.0.30001525",
+    "caniuse-lite": "^1.0.30001532",
     "canvas": "^2.11.2",
-    "core-js": "^3.32.1",
+    "core-js": "^3.32.2",
```

Comment:
Please note that we want to keep the gulpfile-entry in sync with the package version, but looking more closely at https://babeljs.io/docs/babel-preset-env#corejs the current one might be unnecessarily precise? Hence you can probably change https://github.com/mozilla/pdf.js/blob/18a661b6a09fc3b7f4bfaaf4205d760aed61ab3b/gulpfile.mjs#L215 into just
```javascript
{ corejs: "3.32", shippedProposals: true, useBuiltIns: "usage" }, 
```

Reply:
Good point, I missed the extra `corejs` version definition we have there. I have updated it now; I kept the patch version to make sure we have the same version since using 3.32 likely implies 3.32.0 which means we could diverge from the version in `package.json`, so I didn't want to risk that here.